services/audit-service/src/main/java/datawave/microservice/audit/config/AuditProperties.java

```diff
@@ -34,13 +34,14 @@ public Retry getRetry() {
     }
     
     public void setRetry(Retry retry) {
-        this.retry = retry;
+        if (retry != null)
+            this.retry = retry;
```

Comment:
Wouldn't it be better to throw an exception rather than silently doing nothing? If this ever occurred, it would be very difficult to debug if it just silently ignores the set call.